### PR TITLE
Align with the version on donkeycar repo

### PR DESCRIPTION
**Pull request by sctse999**
1. Remove unused code and variable
2. Remove datastr= '' to act as a joystick control
3. Use logging lib
4. Change to autopep8 standard formatting

### CODE REVIEW

File: examples/rc-cpy-final.py [hunk 1]
```diff
@@ -14,182 +15,163 @@
 from digitalio import DigitalInOut, Direction
 from pulseio import PWMOut, PulseIn, PulseOut
 
-## Customisation these variables
+import adafruit_logging as logging
```
Comment (wallarug) [inline examples/rc-cpy-final.py:18]: Does this need a separate library to be installed on the robo HAT?

File: examples/rc-cpy-final.py [hunk 2]
```diff
@@ -14,182 +15,163 @@
...
-	(Testing) Reads RC channel, smooths value and stall prevention (disabled)
-	"""
-        prev = control.value
-	control.channel.pause()
-	for i in range(0, len(control.channel)):
-		#val = control.channel[i]
-                # prevent ranges outside of control space
-		if(val < 1000 or val > 2000):
-			continue
-		# cap maximum acceleration to prevent stall
```
Comment (wallarug) [inline examples/rc-cpy-final.py:61]: I would like to keep this algorithm somewhere, even if it is disabled as it is useful.

File: examples/rc-cpy-final.py (hunk 1 above)
Comment (sctse999) [reply examples/rc-cpy-final.py:18]: Yes, need to put a library on the lib folder

File: examples/rc-cpy-final.py (hunk 2 above)
Comment (sctse999) [reply examples/rc-cpy-final.py:61]: this is not changed i believe. maybe just re-formatting causing it looks like it is deleted.

**Comment by wallarug** [comment]
Guide has been updated to include comment about Adafruit Logging.

https://www.hackster.io/wallarug/autonomous-cars-with-robo-hat-mm1-8d0e65

**Comment by wallarug** [comment]
Updated changes in the description.  Accepting all changes.

**Pull request merged**